.mailmap

```diff
@@ -0,0 +1,7 @@
+Aleksa Sarai <asarai@suse.de> <cyphar@cyphar.com>
+Brandon Philips <brandon.philips@coreos.com> <brandon@ifup.org>
+Doug Davis <dug@us.ibm.com> <duglin@users.noreply.github.com>
+Sergiusz Urbaniak <sur@coreos.com> <sergiusz.urbaniak@gmail.com>
+Stephen J Day <stephen.day@docker.com> <stevvooe@users.noreply.github.com>
+xiekeyang <xiekeyang@huawei.com> <keyang.xie@gmail.com> <xiekeyang@users.noreply.github.com>
```

Comment:
I don't think this two-commit-emails syntax is supported (I don't see it discussed in `git-shortlog(1)` anyway).  So I'm getting:

```
$ git --version
git version 2.1.0.60.g85f0837
$ git describe
v0.2.0-43-gb76b86e
$ git shortlog -se | grep keyang
    21  xiekeyang <xiekeyang@huawei.com>
    22  xiekeyang <xiekeyang@users.noreply.github.com>
```

I think you want two entries:

```
xiekeyang <xiekeyang@huawei.com> <keyang.xie@gmail.com>
xiekeyang <xiekeyang@huawei.com> <xiekeyang@users.noreply.github.com>
```